### PR TITLE
[3502] Add Code Climate Badges to README

### PR DESCRIPTION
**Pull request by aliuk2012**
### Changes proposed in this pull request
- Adds Code Climate badge
- Adds missing Azure badge
### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product review

### CODE REVIEW

**Comment by aliuk2012** [comment]
[Add Code Climate Badges to all our repo README](https://trello.com/c/v0phPiCR/3502-add-code-climate-badges-to-all-our-repo-readme)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_apis/build/status/Find/find-teacher-training?branchName=master)](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build/latest?definitionId=296&branchName=master)
+[![Maintainability](https://api.codeclimate.com/v1/badges/f97679a5f6ddaa3f8981/maintainability)](https://codeclimate.com/github/DFE-Digital/find-teacher-training/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/f97679a5f6ddaa3f8981/test_coverage)](https://codeclimate.com/github/DFE-Digital/find-teacher-training/test_coverage)
```
Comment (TobyRet) [inline README.md:3]: Does this PR also fix the broken `test coverage` badge in the header?

Comment (aliuk2012) [reply README.md:3]: No it doesn't fix it. It just adds the badges to the readme so you don't need a browser extension to see the status

**Comment by aliuk2012** [comment]
@defong No it doesn't add/fix it. It just adds the badges to the readme so you don't need a browser extension to see the status. I'm working on fixing that issue. It currently looks like Find (Rails version) has never used code climate but API/publish has had code climate integrated before.

**Pull request merged**